plugins/unmanaged_files/unmanaged_files_model.rb

```diff
@@ -73,6 +73,14 @@ def compare_with(other)
     comparison.push(common)
   end
 
+  def has_metadata?
+    @metadata ||= has_metadata || elements.any?(&:user)
```

Comment:
Why do we need this `elements.any?(&:user)` part? The user attribute should only be there when the metadata was extracted (and thus `has_metadata` is true) anyway, no?